src/NuGet.Core/NuGet.Packaging/LocalPackageArchiveDownloader.cs

```diff
@@ -154,7 +154,7 @@ public void Dispose()
             {
                 await _sourceStream.Value.CopyToAsync(
                     destination,
-                    bufferSize: 4096,
+                    bufferSize: 8192,
```

Comment:
Should be a constant. Also it'd be great adding a comment saying this value is optimized and should not be changed.